doc/data/messages/c/consider-using-join/bad.py

```diff
@@ -0,0 +1,6 @@
+def fruits_to_string():
```

Comment:
Could have added a more extensive example of building a string with a separator, but deemed this overly complicated for the illustration.